runtime/onert/backend/acl_cl/ConstantInitializer.cc

```diff
@@ -96,6 +101,45 @@ void ConstantInitializer::visit(const ir::operation::SpaceToBatchND &node)
   }
 }
 
+void ConstantInitializer::visit(const ir::operation::Reverse &node)
+{
+  const auto &output_index = node.getOutputs().at(0);
+
+  const auto &input_index = node.getInputs().at(ir::operation::Reverse::Input::INPUT);
+  const auto &input_obj = _operands.at(input_index);
+
+  const auto &axis_index = node.getInputs().at(ir::operation::Reverse::Input::AXIS);
+  const auto &axis_obj = _operands.at(axis_index);
+
+  const auto ifm_rank = input_obj.shape().rank();
+  const auto frontend_layout = this->_current_op_seq_layout;
+
+  auto output_tensor = this->_tensor_reg->getITensor(output_index);
+  const auto backend_layout = output_tensor->layout();
+
+  if (axis_obj.isConstant())
+  {
+    _init_map[axis_index] = [&](const ir::Operand &operand, backend::ITensor &obj) {
+      assert(operand.data());
```

Comment:
I think that it should be not capture by reference but capture by value.

Reply:
I think `operand` would be okay since `operand` is a parameter, but not captured.
But `ifm_rank`, `frontend_layout`, and `backend_layout` below are dangerous. They should be captured by value.

Reply:
I agree with this opinion